### ISSUE TITLE
Build is failed and I can't use this plugin as well

### ISSUE THREAD

**Issue by ozbillwang** [drone-plugins/drone-hg#13]
With below pipeline

```
$ cat .drone.yml

clone:
  hg:
    image: plugins/drone-hg
    path: bitbucket.org/foo/bar
```
I got the error:

```
Drone Mercurial Plugin built from 70dad5c
panic: EOF
goroutine 1 [running]:
panic(0x5281a0, 0xc82000a110)
	/usr/local/go/src/runtime/panic.go:481 +0x3e6
github.com/drone-plugins/drone-hg/vendor/github.com/drone/drone-plugin-go/plugin.MustParse()
	/drone/src/github.com/drone-plugins/drone-hg/vendor/github.com/drone/drone-plugin-go/plugin/param.go:129 +0x59
main.main()
	/drone/src/github.com/drone-plugins/drone-hg/main.go:44 +0x54a
```

Please help.

**Comment by ozbillwang** [comment]
used the wrong image `plugins/drone-hg`, have to wait the image `plugins/hg` to be available  (#15)

**Comment by tboerger** [comment]
This image should be available now
